package.json

```diff
@@ -22,14 +22,19 @@
   "dependencies": {
     "@actions/core": "^1.2.6",
     "@actions/github": "^2.1.1",
+    "@octokit/rest": "^16.43.1",
+    "@slack/web-api": "^6.0.0",
     "applicationinsights": "^1.7.6",
     "axios": "^0.21.1",
+    "azure-storage": "^2.10.3",
+    "stream-buffers": "^3.0.2",
```

Comment:
Installing azure storage for every run is a bit expensive (adds ~1s to the action runtime, which is small but adds up). For workflows that need it I usually install it in the workflow run script, then keep in in dev dependencies here. (for instance deep-classifier-scraper.yml in the vscode repo, https://github.com/microsoft/vscode/blob/c7fa31fc926a006865462fa9bba5760364434bcc/.github/workflows/deep-classifier-scraper.yml#L20)

Note we aalready use blob storge, and it's a bit nicer to work with than file storage (no stream-buffers needed), so if you can keep the drop in a blob instead that's a bit nicer. Not a big deal either waay. Some code for this is in classifier-deep/blobStorage/index.ts

Reply:
Done. One downside is that installing it only when needed makes it harder to reuse the action. E.g., vscode-remote-containers now also needs to know which package and version to install: https://github.com/microsoft/vscode-remote-containers/blob/2a039da40ca99da4f053eab6aef80281c0f4f0a4/.github/workflows/build-chat.yml#L26

Reply:
I agree it's not great, a better solution might be to have a package.json in eaach aaction's directory. I will investigate that as a debt item